app/views/admin/statistics_announcements/show/_main.html.erb

```diff
@@ -0,0 +1,143 @@
+<div class="app-view-summary__main">
+  <section class="app-view-summary__section">
+    <p class="govuk-body-lead">
+      <%= @statistics_announcement.summary %>
+    </p>
+
+    <%=
+      render "govuk_publishing_components/components/summary_list", {
+        borderless: true,
+        items: [
+          { field: "Type", value: @statistics_announcement.display_type },
+          { field: "Organisations", value: @statistics_announcement.organisations.map(&:name).to_sentence },
+        ],
+      }
+    %>
+
+    <%= render "components/inset_prompt", {
+      error: true,
+      description: "Statistics release cancelled - #{@statistics_announcement.cancellation_reason}.",
+    } if @statistics_announcement.cancelled? %>
+  </section>
+
+  <section class="app-view-summary__section">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Release and delivery date",
+      heading_level: 2,
+      font_size: "l",
+      margin_bottom: 3,
+    } %>
+    <p class="govuk-body">
+      <%= link_to("Change dates",
+                  new_admin_statistics_announcement_change_path(@statistics_announcement),
+                  class: "govuk-link",
+                  data: {
+                    module: "gem-track-click",
+                    "track-category": "button-clicked",
+                    "track-action": "#{@statistics_announcement.model_name.singular.dasherize}-button",
+                    "track-label": "Change dates",
+                  }) %>
+    </p>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Release date",
+      heading_level: 3,
+      font_size: "m",
+      margin_bottom: 2,
+    } %>
+    <% if @statistics_announcement.cancelled? %>
+      <p>
```

Comment:
can we add `govuk-body"` to the para here and remove it from the spans?

Reply:
✅